### DISCUSSION TITLE
Image download

### DISCUSSION THREAD

**Discussion by vaasal** [php-webdriver/php-webdriver#1039] [Q&A]
Why there isnt any method to downlad an image? I've serarch but I don't find nothing easy an nothing in php.

**Comment by OndraM** [comment]
Hi, you can read the image URL (for example `$imageUrl = $imgElement->getAttribute('src');`) and then you can use native PHP functions to download the file. https://stackoverflow.com/a/3938551/464890

**Reply by vaasal** [reply]
Thank you. I knwow how to do this in php, but the problem is I can't download direct the image due the limitation of clouflare. It's the reason I use php-webdriver.
